### DISCUSSION TITLE
Discussions!

### DISCUSSION THREAD

**Discussion by DavidColby** [DavidColby/hotwired_ats_code#21] [Announcements]
👋 Hi folks! Thanks for reading Hotwired ATS. If you run into trouble with the book or have questions about any of the content, please use this discussion board to get help from myself and other readers, when time allows.

**Comment by DebakeDSP94** [comment]
Hi, I found a small error in chapter 3.  In the text you reference "app/controllers/application_controller.rb" but in the code image at the top it says "app/views/controllers/application_controller.rb"  I think most readers would figure it out, but just wanted to let you know.  I'm enjoying the book so far.  Very cool stuff.  Thanks.  
![Screenshot from 2022-09-16 15-44-23](https://user-images.githubusercontent.com/99148484/190730033-72435958-d58e-4ce5-a1a9-e39604b9615c.png)

**Reply by DavidColby** [reply]
Thanks for raising this. It is fixed!

**Reply by DebakeDSP94** [reply]
You are welcome sir.  I have a question. After getting through chapter 6, should I be able to send emails from the app to say, my gmail account?  I have tried and none have come through.  I wasn't quite sure if it is supposed to be fully functional at this point or if another component is needed.  Thanks.

**Reply by DavidColby** [reply]
You won't be able to send emails from the app running locally to your gmail account. Typically, when you are running an application on your local machine, email isn't sent anywhere but can be viewed in the Rails logs or saved to a file for review. For easier testing locally, you can also use a gem like [letter_opener_web](https://github.com/fgrehm/letter_opener_web) to view emails that have been "sent" locally.

To have your Rails app send real email to real email addresses, you need to configure your app to use a mail service, which you would normally do when you have deployed the app to a production environment like Heroku. The [Rails ActionMailer](https://guides.rubyonrails.org/action_mailer_basics.html#action-mailer-configuration) guide is a good starting place for learning about this — production deployment, including send real email is not covered in the book!

**Reply by DebakeDSP94** [reply]
I thought that might be the case, but I wasn't sure if it was that or I had missed something when doing that section.  Thanks.
